src/main/java/com/minelittlepony/IPonyConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.minelittlepony;
+
+import com.minelittlepony.pony.data.PonyLevel;
+
+public interface IPonyConfig {
+    default boolean getSizes() { return true; }
```

Comment:
I'm not a fan of using default values for everything. Either implement it or don't.